### DISCUSSION TITLE
orm-v4.4.0

### DISCUSSION THREAD

**Discussion by medz** [medz/dart-orm#388] [Announcements]
To install Prisma ORM for Dart v4.4.0 run this command:

```bash
dart pub add orm:4.4.0
```

or update your `pubspec.yaml` file:

```yaml
dependencies:
  orm: ^4.4.0
```

## What's Changed

* fix(deps): update dependency webfetch to ^0.0.17 by @renovate in https://github.com/medz/prisma-dart/pull/383
* feat(client): support `createManyAndReturn` model delegate method by @medz in https://github.com/medz/prisma-dart/pull/386

**Full Changelog**: https://github.com/medz/prisma-dart/compare/orm-v4.3.0...orm-v4.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/medz/prisma-dart/releases/tag/orm-v4.4.0'>orm-v4.4.0</a>.</em>

**Comment by Batutankuma** [comment]
error 
Target of URI hasn't been generated: 'package:prisma_dart/src/generated/prisma/prisma_client.g.dart'.
Try running the generator that will generate the file referenced by the URI.dart[uri_has_not_been_generated](https://dart.dev/diagnostics/uri_has_not_been_generated)
